src/pkgcheck/checks/codingstyle.py

```diff
@@ -1118,3 +1118,53 @@ def feed(self, pkg):
             lines.append(lineno)
         if lines:
             yield ExcessiveLineLength(lines=lines, pkg=pkg)
+
+
+class InstallCompressedManpage(results.LineResult, results.Warning):
+    """Compressed manpages are not supported by ``doman`` or ``newman``."""
+
+    def __init__(self, func, **kwargs):
+        super().__init__(**kwargs)
+        self.func = func
+
+    @property
+    def desc(self):
+        return f'line {self.lineno}: compressed manpage {self.line!r} passed to {self.func}'
+
+
+class InstallCompressedInfo(results.LineResult, results.Warning):
+    """Compressed manpages are not supported by ``doinfo`` or ``newinfo``."""
```

Comment:
newinfo doesn't exist!

Reply:
lol - Just shows how much I don't use it 😄 
(FYI, I even have in my system `INSTALL_MASK="/usr/share/info/"`)